book/src/super-sql/operators/from.md

```diff
@@ -18,7 +18,7 @@ a query and transmits that data to its output.
 
 It has two forms:
 * a `from` pipe operator with dataflow scoping as described here, or
-* a [`FROM`](../sql/from.md) SQL clause with relational scoping.
+* a [SQL `FROM` clause](../sql/from.md) with relational scoping.
```

Comment:
Similar to my previous comment about a convention for hyperlinking to pipeline element docs, it seems like a reasonable convention would be to hyperlink to `SQL <something> clause`, since lots of SQL resources I see out there (e.g., [w3schools](https://www.w3schools.com/sql/sql_where.asp)) seem to lead with the "SQL".